js/formatXML.js

```diff
@@ -0,0 +1,51 @@
+// eslint-disable-next-line no-unused-vars
+function formatXml(input, indent) {
+  indent = indent || "  "; //you can set/define other ident than tabs
```

Comment:
Does `formatXML` belong to source code?  

You have moved this code here because is shared between demo and tests, but It won't be added to the final bundle, right?

Reply:
Didn't think about this. Will check.